src/includes/enriching-events/attach-screenshots/unity.mdx

```diff
@@ -0,0 +1,13 @@
+The **Attach Screenshot** options can be found on the `Enrichment` tab in the Editor Configuration Window.
+
+![Screenshots Thumbnail Example](unity-screenshots.png)
+
+<Note>
+
+Note that capturing a screenshot in the Unity SDK requires running on the main thread.
```

Comment:
Making a wording change here to try to clarify, but if I've done so incorrectly, let me know.

```suggestion
Capturing a screenshot in Unity requires running the SDK on the main thread.
```